### PR TITLE
kotlin wizard

### PR DESCRIPTION
**Pull request by bruno-garcia**
@PeloWriter @maciejwalkowiak @marandaneto thoughts?

### CODE REVIEW

File: src/wizard/kotlin/index.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+name: Kotlin
+doc_link: https://docs.sentry.io/platforms/kotlin/
+support_level: production
+type: language
+---
+
+<Alert level="info">
+    Sentry supports Kotlin for the JVM and Android. Here we'll cover the JVM scenario. The Android project type has Kotlin code snippets. At its core, Sentry for Java provides a raw client for sending events to Sentry. If you use <strong>Spring Boot</strong>, <strong>Spring</strong>,<strong> Logback</strong>, or <strong>Log4j2</strong>, we recommend visiting our <a href="https://docs.sentry.io/platforms/java/">Sentry Java</a> documentation for installation instructions.
```
Comment (PeloWriter) [inline src/wizard/kotlin/index.md:9]:
```suggestion
Sentry supports Kotlin for both JVM and Android. This wizard guides you through set up in the JVM scenario. 

If you're interested in <strong>Android</strong>, head over to the <a href="https://docs.sentry.io/platforms/android/">Getting Started</a> for that SDK instead. At its core, Sentry for Java provides a raw client for sending events to Sentry. If you use <strong>Spring Boot</strong>, <strong>Spring</strong>,<strong> Logback</strong>, or <strong>Log4j2</strong>, head over to our <a href="https://docs.sentry.io/platforms/java/">Getting Started for Sentry Java</a>.
```

File: src/wizard/kotlin/index.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+} catch (e: Exception) {
+  Sentry.captureException(e)
+}
+```
+
+If you're new to Sentry, use the email alert to access your account and complete a product tour.
+
+If you're an existing user and have disabled alerts, you won't receive this email.
+
+### Measure performance
```
Comment (PeloWriter) [inline src/wizard/kotlin/index.md:75]:
```suggestion
### Measure Performance
```

**Review by PeloWriter — APPROVED** [review]

File: src/wizard/kotlin/index.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+doc_link: https://docs.sentry.io/platforms/kotlin/
+support_level: production
+type: language
+---
+
+<Alert level="info">
+Sentry supports Kotlin for both JVM and Android. This wizard guides you through set up in the JVM scenario. 
+
+If you're interested in <strong>Android</strong>, head over to the <a href="https://docs.sentry.io/platforms/android/">Getting Started</a> for that SDK instead. At its core, Sentry for Java provides a raw client for sending events to Sentry. If you use <strong>Spring Boot</strong>, <strong>Spring</strong>,<strong> Logback</strong>, or <strong>Log4j2</strong>, head over to our <a href="https://docs.sentry.io/platforms/java/">Getting Started for Sentry Java</a>.
+</Alert>
```
Comment (marandaneto) [inline src/wizard/kotlin/index.md:12]:
```suggestion
If you're interested in <strong>Android</strong>, head over to the <a href="https://docs.sentry.io/platforms/android/">Getting Started</a> for that SDK instead. At its core, Sentry for Java provides a raw client for sending events to Sentry. If you use <strong>Spring Boot</strong>, <strong>Spring</strong>, <strong>Logback</strong>, <strong>JUL</strong>, or <strong>Log4j2</strong>, head over to our <a href="https://docs.sentry.io/platforms/java/">Getting Started for Sentry Java</a>.
</Alert>
```

File: src/wizard/kotlin/index.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+} catch (e: Exception) {
+  transaction.throwable = e
+  transaction.status = SpanStatus.INTERNAL_ERROR
+  throw e
+} finally {
+  transaction.finish();
+}
+```
+
+For more information about the API and automatic instrumentations included in the SDK, [visit the docs](https://docs.sentry.io/platforms/java/performance/instrumentation/custom-instrumentation/).
```
Comment (marandaneto) [inline src/wizard/kotlin/index.md:98]:
```suggestion
For more information about the API and automatic instrumentations included in the SDK, [visit the docs](https://docs.sentry.io/platforms/java/performance/).
```

it mentions `automatic instrumentations` but it links to `custom-instrumentation` so I'd prefer linking to the root instead

**Pull request merged**